presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -168,58 +168,73 @@ Configuration Properties
-                                                   Otherwise, it will be ignored.
-
-``iceberg.hadoop.config.resources``                The path(s) for Hadoop configuration resources.
-                                                   ``Example: /etc/hadoop/conf/core-site.xml.`` This property
-                                                   is required if the iceberg.catalog.type is hadoop. Otherwise,
-                                                   it will be ignored.
-
-``iceberg.max-partitions-per-writer``              The Maximum number of partitions handled per writer.          ``100``
-
-``iceberg.minimum-assigned-split-weight``          A decimal value in the range (0, 1] is used as a minimum      ``0.05``
-                                                   for weights assigned to each split. A low value may improve
-                                                   performance on tables with small files. A higher value may
-                                                   improve performance for queries with highly skewed
-                                                   aggregations or joins.
-
-``iceberg.enable-merge-on-read-mode``              Enable reading base tables that use merge-on-read for          ``true``
-                                                   updates.
-
-``iceberg.delete-as-join-rewrite-enabled``         When enabled, equality delete row filtering is applied        ``true``
-                                                   as a join with the data of the equality delete files.
-================================================== ============================================================= ============
+======================================================= ============================================================= ============
+Property Name                                           Description                                                   Default
+======================================================= ============================================================= ============
+``hive.metastore.uri``                                  The URI(s) of the Hive metastore to connect to using the
+                                                        Thrift protocol. If multiple URIs are provided, the first
+                                                        URI is used by default, and the rest of the URIs are
+                                                        fallback metastores.
+                                                        Example: ``thrift://192.0.2.3:9083`` or
+                                                        ``thrift://192.0.2.3:9083,thrift://192.0.2.4:9083``
+                                                        This property is required if the
+                                                        ``iceberg.catalog.type`` is ``hive``. Otherwise, it will
+                                                        be ignored.
+
```

Comment:
Consider moving the entry for ``iceberg.catalog.type`` to immediately follow the entry for ``hive.metastore.uri``, because the two are related. 

Of course, ``hive.metastore.uri`` is not the only one related to ``iceberg.catalog.type``. Maybe a good order for this table could be: 
``hive.metastore.uri``
``iceberg.catalog.type`` 
``iceberg.catalog.warehouse``
``iceberg.catalog.cached-catalog-num``
``iceberg.hadoop.config.resources``
then
``iceberg.file-format``
``iceberg.compression-codec``
then 
``iceberg.max-partitions-per-writer``
and the rest. 

Maybe move ``iceberg.file-format`` and ``iceberg.compression-codec`` to the beginning (if those are very important to the user), then the block of catalog type-connected properties, then ``iceberg.max-partitions-per-writer`` and the rest. 

My idea is to group the block of catalog type-connected properties together, and you would know best the relative importance of that block and the other properties in the table. Does this make sense?